server/src/main/java/org/elasticsearch/action/admin/cluster/stats/AnalysisStats.java

```diff
@@ -34,11 +36,17 @@
 import java.util.Objects;
 import java.util.Set;
 
+import static org.elasticsearch.TransportVersion.V_8_500_040;
+
 /**
  * Statistics about analysis usage.
  */
 public final class AnalysisStats implements ToXContentFragment, Writeable {
 
+    private static final String[] SYNONYM_TYPES = { "synonyms_set", "synonyms_path", "synonyms" };
```

Comment:
should we remove this for this PR, as it is not used.

Reply:
Thanks for the catch - done!